src/platform/plugins/shared/dashboard/public/dashboard_app/url/bwc/extract_options.ts

```diff
@@ -19,23 +19,23 @@ export function extractOptions(state: { [key: string]: unknown }): Partial<Dashb
   const options: Partial<Writable<DashboardOptions>> = {};
```

Comment:
I haven't updated this section or the above section where the `options` object is extracted wholesale because this PR and the PR that made options optional are both going into 9.3.0. This means

- there will be no URLs passed in where the options are spread directly into Dashboard state and in `snake_case`
- there will be no URLs passed in where the options are under the `options` key but are `camelCase`.